include/mbgl/storage/default_file_source.hpp

```diff
@@ -121,11 +121,6 @@ class DefaultFileSource : public FileSource {
      * Eviction works by removing the least-recently requested resources not also required
```

Comment:
It's not clear from the documentation alone: is `VACUUM` triggered by the eviction invoked by this API?